pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: mysql1
-version: 0.18.1
+version: 0.19.0-null-safety.0
```

Comment:
Reading the migration guide I think we can just publish this as `0.19.0`

https://dart.dev/null-safety/migration-guide#package-version